apps/hash-graph/Makefile.toml

```diff
@@ -34,7 +52,7 @@ cwd = "${REPO_ROOT}"
 
```

Comment:
(for deployment-down task)
do we perhaps want to pass `"${@}"` at the end of the args here to be able to do something like `cargo make deployment-down -v` to wipe volumne? We could add it in `*-up` too.

We already do this in `tasks.build-docker`

Reply:
Discussed offline, we tried this before and it conflicts with CI params